qrscanner/src/main/java/io/gnosis/safe/qrscanner/QRCodeScanActivity.kt

```diff
@@ -8,9 +8,9 @@ import android.os.Bundle
 import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.app.AppCompatActivity
 import androidx.fragment.app.Fragment
-import kotlinx.android.synthetic.main.screen_scan.*
 import pm.gnosis.svalinn.utils.ethereum.ERC67Parser
 import pm.gnosis.utils.asEthereumAddress
+import kotlinx.android.synthetic.main.screen_scan.*
```

Comment:
did you want to remove synthetic?

Reply:
I could not remove it from QRScanner because it would break. We need a separate ticket. Also for migrating it out of its own module. I will make such a ticket.